src/widget/wlibrarysidebar.cpp

```diff
@@ -239,12 +239,36 @@ void WLibrarySidebar::keyPressEvent(QKeyEvent* event) {
     // TODO(XXX) Should first keyEvent ensure previous item has focus? I.e. if the selected
     // item is not focused, require second press to perform the desired action.
 
-    // make the selected item the navigation starting point
+    SidebarModel* sidebarModel = qobject_cast<SidebarModel*>(model());
+    QModelIndexList selectedIndices = selectionModel()->selectedRows();
+    if (sidebarModel && !selectedIndices.isEmpty()) {
+        QModelIndex index = selectedIndices.at(0);
+        if (event->matches(QKeySequence::Delete) || event->key() == Qt::Key_Backspace) {
+            sidebarModel->clear(index);
+            return;
+        }
+        if (event->matches(QKeySequence::Cut)) {
+            sidebarModel->cut(index);
+            return;
+        }
+        if (event->matches(QKeySequence::Copy)) {
+            sidebarModel->copy(index);
+            return;
+        }
+        if (event->matches(QKeySequence::Paste)) {
```

Comment:
I think we should remove all these shortcut from the side-bar. The only reasonable shortcut is this. 
Select a playlist and drop a track to it.

Reply:
Yes, I left all these methods as virtuals without implementations, during development, to decide what made sense to implement on the go. It might be better to remove them now altogether, and add them back if needed.

Reply:
Would you prefer that?

Reply:
Yes. We should not allow to "remotely" copy or delete a track when the focus is on the sidebar. The only thing that we may allow is Ctrl+V analog to doping a track to the playlist in the sidebar.

Reply:
I cant reproduce the behaviour I might have seen. So it was probably a wrong observation.

Reply:
Ok, but still, I will remove the code that only can lead to empty methods.